jsk_perception/docker/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# Author: Yoshiki Obinata <obinata@jsk.imi.i.u-tokyo.ac.jp>
+#
+
+# api directories
+OFAPROJECT = ofa
+CLIPPROJECT = clip
+# image names
+OFAIMAGE = jsk-ofa-server
+CLIPIMAGE = jsk-clip-server
+# commands
+BUILDIMAGE = docker build
+REMOVEIMAGE = docker rmi
+# configuration directories
+CONFIGDIR = $(shell echo ${HOME}/.jsk_vil_apis)
+PARAMDIR = $(CONFIGDIR)/params
+# Download files
+PARAMURLS = parameter_urls.txt
+# OFA parameters
+OFAPARAMFILES = $(foreach param, $(OFAPARAMS), $(PARAMDIR)/$(param))
+
+# TODO check command wget exists, nvidia-driver version
+
+# TODO add clip
+download:
+	@if [ ! -d $(PARAMDIR) ]; then\
+		mkdir -p $(PARAMDIR); \
+	fi
+	wget -c -i $(PARAMURLS) -P $(PARAMDIR)
```

Comment:
add `touch download` in order to skip running `wget` when it already downloaded the param files. 
or set hidden file such as `$(PARAMDIR)/.download` as a make target